### ISSUE TITLE
Is this provider compatible with 0.12?

### ISSUE THREAD

**Issue by jakubgs** [terraform-providers/terraform-provider-alicloud#1188]
Hey!
A `0.12` candidate release has been pulished:
https://www.hashicorp.com/blog/releasing-terraform-0-12-0-rc1
I was wondering if this provider already supports it or not.

**Comment by programmer04** [comment]
As I tested unfortunately no. For [allicloud provider v1.43.0](https://github.com/terraform-providers/terraform-provider-alicloud/releases/tag/v1.43.0) and [Terraform v0.12.0](https://github.com/hashicorp/terraform/releases/tag/v0.12.0) I got:
```bash
Error: Failed to instantiate provider "alicloud" to obtain schema: Incompatible API version with plugin. Plugin version: 4, Client versions: [5]
```
Any ETA?
@xiaozhu36 @xh4n3

**Comment by xiaozhu36** [comment]
HI . @jakubgs @programmer04 I need to evaluate supporting Terraform v0.12.0. Please be patient. Thanks a lot.

**Comment by tdfacer** [comment]
I'm also very interested in this.  Thank you!

**Comment by veysiertekin** [comment]
Hi @xiaozhu36,

Thank you for all your efforts. We are really looking forward to this feature to migrate 0.12. Are there any updates yet?

**Comment by bittopaz** [comment]
@xiaozhu36 The OSS backend support is released in 0.12.2, which is very exciting. Really looking forward seeing compatibility with 0.12.

**Comment by xiaozhu36** [comment]
HI all, I am working on it. Please be patient. Thanks a lot.

**Comment by allspace** [comment]
@xiaozhu36  Waiting for your good news :-)

**Comment by veysiertekin** [comment]
Hi @xiaozhu36 ,

Is there any updates for the compatibility :)

**Comment by xiaozhu36** [comment]
Hi everyone, we are working on it. We plan the provider release 1.50.0 supports terraform 0.12.0 and it will be published in this week or the next week. Please be patient.

**Comment by tdfacer** [comment]
Thanks for the update!

**Comment by xiaozhu36** [comment]
HI all, the latest release 1.50.0 has been published and it has supported `0.12`. Please check it.
